it/src/test/java/org/corfudb/universe/scenario/OneNodeDownIT.java

```diff
@@ -44,15 +45,14 @@ public void oneNodeDownTest() {
             }
 
             testCase.it("Should stop one node and then restart", data -> {
-                CorfuServer server0 = corfuCluster.getFirstServer();
+                CorfuServer server2 = corfuCluster.getServerByIndex(2);
 
                 // Stop one node and wait for layout's unresponsive servers to change
-                server0.stop(Duration.ofSeconds(10));
-                waitForUnresponsiveServersChange(size -> size == 1, corfuClient);
-
-                // Verify layout, unresponsive servers should contain only the stopped node
-                Layout layout = corfuClient.getLayout();
-                assertThat(layout.getUnresponsiveServers()).containsExactly(server0.getEndpoint());
+                // We choose the node with largest endpoint to prevent jittering
+                // TODO: rotate the node to stop to fully test jittering and deadlock cases
+                server2.stop(Duration.ofSeconds(10));
+                waitForLayoutChange(layout -> layout.getUnresponsiveServers()
```

Comment:
It's a refactoring but what was the problem that it is trying to fix?

Reply:
This is a better way to do assert, to eliminate any possibility of jittering.